components/board/Board.tsx

```diff
@@ -0,0 +1,37 @@
+import Image from 'next/image';
+import styles from '@/components/board/Board.module.scss';
+import formatDate from '@/utils/formatDate';
+import Like from './Like';
+import ProfileImage from '../ProfileImage';
+import Kebab from '../Kebab';
+
+function Board({ article }: TArticle) {
+  const { id, title, content, image, writer, likeCount, createdAt, updatedAt } =
+    article;
+  const { nickname } = writer;
```

Comment:
```suggestion
  const { id, title, content, image, writer: {nickname}, likeCount, createdAt, updatedAt } =
    article;
```
writer가 따로 사용되지 않는다면 구조분해할당으로도 가능해요!